1_fetch.R

```diff
@@ -160,30 +160,45 @@ p1_targets_list <- list(
     read_csv(p1_sntemp_inputs_outputs_csv,show_col_types = FALSE)
   ),
 
-  # Download NLCD datasets 
+  # Read in all nlcd data from 2001-2019 
+  # Note: NLCD data must already be downloaded locally and manually placed in NLCD_LC_path ('1_fetch/in/NLCD_final/')
   tar_target(
-    p1_NLCD_data_zipped, 
-    download_NHD_data(sb_id = sb_ids_NLCD,
+    p1_NLCD_LC_data,
+    {if(length(list.files(NLCD_LC_path)) == 0){
+        stop(paste0('No NLCD LC data in ', NLCD_LC_path,'. Please move the NLCD .txt files to this location.'))
+        }
```

Comment:
I think this should be part of the `read_subset_LC_data` instead. Wouldn't we always want this check before that function runs?

Reply:
yes agreed. added it in in commit https://github.com/USGS-R/drb-inland-salinity-ml/pull/97/commits/2b44ddbc7c46af063b118bfb0fc1a071dc41d1b4